app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if len(command) == 3 and command[0] == "cp":
+        with (
+            open(command[1], "r") as read_file,
+            open(command[2], "w") as write_file
+        ):
+            for line in read_file.readlines():
+                write_file.write(line)
```

Comment:
```suggestion
            write_file.write(read_file.read())
```
no need to use loop here